### PR TITLE
Completed subscriber sign-up service with proper mapping to counties with multi-selection.

### PR DESCRIPTION
**Pull request by MurasatoNaoya**
The subscriber sign-up functionality is now complete, and updates the subscriber_county_assignment table as intended. The service also account for the multi-selection of counties for a single subscriber, making several subscriber to county mappings using bulk insertions.

Dashboard formatting wise, distinct sections of the dashboard have also been separated into clear sections. Namely 'Home', 'Weather', 'Stellarium Integration' and 'Subscriber Signup'. These are still placeholders and will likely still be changed in the near future.

This closes: #239 #230 #220 #235 #244

### CODE REVIEW

**Comment by MurasatoNaoya** [comment]
A very basic script for bulk reading from the database has also been added, as well as a function for establishing a connection to our RDS. This is more for minimising the amount of boilerplate code in our main dashboard.py file.

File: dashboard/load_dashboard_data.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+
+def connect_to_db():
+    '''Returns a psycopg2 connection object'''
+
+    return psycopg2.connect(
+        host=os.getenv('DB_HOST'),
+        database=os.getenv('DB_NAME'),
+        user=os.getenv('DB_USER'),
+        password=os.getenv('DB_PASSWORD')
```
Comment (CalebMathers) [inline dashboard/load_dashboard_data.py:18]: Could include port here too just to be sure

**Review by CalebMathers — APPROVED** [review]

File: dashboard/dashboard.py [hunk 2]
```diff
@@ -96,62 +254,179 @@ def generate_stellarium_url(lat, lon, dt):
...
 
-        elif not email and not phone:
-            st.error('Please provide either an email or a phone number.')
+        # Either a phone number or emails as to be provided.
+        elif not user_email and not user_phone:
+            st.error('🛑 Please provide either an email or a phone number.')
 
-        elif email and not validate_email(email):
+        elif user_email and not validate_email(user_email):
+            st.error('🛑 Invalid email format. Please enter a valid email.')
```
Comment (AlexOBarnes) [inline dashboard/dashboard.py:330]: Good error handling

File: dashboard/dashboard.py [hunk 3]
```diff
@@ -96,62 +254,179 @@ def generate_stellarium_url(lat, lon, dt):
...
+                        execute_values(cursor, '''
+                            INSERT INTO subscriber_county_assignment (subscriber_id, county_id)
+                            VALUES %s;
+                        ''', county_tuples)
+
+
+                    # Commiting the valid user information to the 'subscriber' table in the RDS,
+                    # while also mapping that subscriber instance to their associated county ID.
+                    conn.commit()
+                st.success('✅ Your subscription has been added successfully!')
```
Comment (AlexOBarnes) [inline dashboard/dashboard.py:368]: Good logging

**Review by AlexOBarnes — APPROVED** [review]

**Pull request merged**